variables.tf

```diff
@@ -61,3 +61,7 @@ variable "wafFileUploadLimit" {
 }
 
 variable "aks_subscription_id" {}
+
+variable "sku" {
```

Comment:
better to just hardcode it, unless there's a need to have a different value in different environments